### PR TITLE
Adding CSP rules to allow Pontoon to talk to the app

### LINKED ISSUE #1954: Add CSP rules for pontoon to communicate with the app for iframe related
We will have to allow Pontoon to communicate with our server for front-end localization.

### PR DESCRIPTION
**Pull request by alicoding**
Closes #1954

@cadecairos R?

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -88,7 +88,8 @@ app.use(helmet.contentSecurityPolicy({
   directives: {
     defaultSrc: [
       'www.youtube.com',
-      'https://public.etherpad-mozilla.org'
+      'https://public.etherpad-mozilla.org',
+      'https://pontoon.mozilla.org'
```
Comment (cadecairos) [inline app.js]: Why'd you add it to every directive?

Surely pontoon doesn't need to be allowed by default.

File: app.js [hunk 2]
```diff
@@ -99,20 +100,23 @@ app.use(helmet.contentSecurityPolicy({
...
-      'https://www.mozilla.org'
+      'https://www.mozilla.org',
+      'https://pontoon.mozilla.org'
     ],
     fontSrc: [
       '\'self\'',
       'fonts.googleapis.com',
-      'fonts.gstatic.com'
+      'fonts.gstatic.com',
+      'https://pontoon.mozilla.org'
```
Comment (cadecairos) [inline app.js]: do we load fonts from pontoon?

File: app.js [hunk 3]
```diff
@@ -99,20 +100,23 @@ app.use(helmet.contentSecurityPolicy({
...
     ],
     styleSrc: [
       '\'self\'',
       '\'unsafe-inline\'',
       'https://www.google.com',
       'fonts.googleapis.com',
       'https://api.tiles.mapbox.com',
-      'https://s.ytimg.com'
+      'https://s.ytimg.com',
+      'https://pontoon.mozilla.org'
```
Comment (cadecairos) [inline app.js]: and style?

File: app.js [hunk 4]
```diff
@@ -127,6 +131,7 @@ app.use(helmet.contentSecurityPolicy({
       '*.mywebmaker.org',
       '*.makes.org',
       'bitly.mofoprod.net',
+      'https://pontoon.mozilla.org',
```
Comment (cadecairos) [inline app.js]: and images?

File: app.js (hunk 2 above)
Comment (alicoding) [inline app.js]: Yup we do

File: app.js (hunk 3 above)
Comment (alicoding) [inline app.js]: Yup

File: app.js (hunk 4 above)
Comment (alicoding) [inline app.js]: Yup

File: app.js (hunk 1 above)
Comment (alicoding) [inline app.js]: Pontoon needs everything in order to function properly...

Comment (cadecairos) [inline app.js]: wat.

Comment (alicoding) [inline app.js]: Well... Its in an iframe and trying to load everything for users to translate the site and it needs to display everything correctly the same way on production.

**Comment by cadecairos** [comment]
R+

**Pull request merged**